Cargo.toml

```diff
@@ -23,7 +23,9 @@ hex = { version = "0.2", package = "hex-conservative" }
 log = "^0.4"
 minreq = { version = "2.11.0", features = ["json-using-serde"], optional = true }
 reqwest = { version = "0.11",  features = ["json"], default-features = false, optional = true }
-async-std = { version = "1.13.0", optional = true }
+
+# default async runtime
+tokio = { version = "1.40", features = ["time"], optional = true }
```

Comment:
I had to change this to version 1.38 in order to build on MSRV following the pinned deps in the README, not sure if it matters here